src/platform/utilities/data/clone.js

```diff
@@ -6,19 +6,17 @@
  */
 export default function clone(object) {
   /* eslint-disable no-prototype-builtins */
-  switch (typeof object) {
-    case 'object':
-      if (Array.isArray(object)) {
-        return object.slice();
-      } else if (Set.prototype.isPrototypeOf(object)) {
-        return new Set(object);
-      } else if (Map.prototype.isPrototypeOf(object)) {
-        return new Map(object);
-      }
-      // Probably just ye olde object
-      return Object.assign({}, object);
-    default:
-      throw new Error(`Unknown type in clone: ${typeof object}`);
+  if (typeof object === 'object') {
```

Comment:
I see this `clone` being imported into the following files:

- `src/platform/utilities/data/set.js`
- `src/platform/utilities/data/cloneDeep.js`
- `src/platform/utilities/data/index.js`
- `src/platform/utilities/data/unset.js`